vignettes/pkgdown.Rmd

```diff
@@ -147,10 +149,8 @@ See `?build_news` for more customisation options including how to:
 
 If you use GitHub, there are two ways to publish your site on GitHub Pages:
```

Comment:
Since we're now only recommending one way (which is good!), this sentence needs to be tweaked, and I think we can get rid of the bullets.

Reply:
I also made changes in README to reflect that everywhere!